docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+-f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+numpy
+torch_nightly
```

Comment:
Do we really want to have dependency on torch_nightly ? Shouldn't we agree on a release instead ?

Reply:
Fixed. Now using pinned pytorch version 1.3.1 with corresponding torchvision version.